plutus-playground-client/update-deps.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
This is literally the same script, so if you put it in `dev.scripts` like I suggested you can just use it in both places.

Reply:
The blocker there is building plutus on OSX generally. Until that works, I don't think we can move these scripts into nixspace.

I'm going to bite the bullet and see if enough updates have trickled down the pipeline to finally fix the underlying problem. 🤞

Reply:
I thought the problem only kicked in relatively late in the pipeline, i.e. when building the purescript apps? Building just these scripts should hopefully be less painful?

Reply:
Sadly not. Building the server fails on the latest OSX.

Reply:
But building the *script* shouldn't be a problem. To be clear, I mean making a derivation which outputs a properly wrapped version of this script, as with `fixStylishHaskell`. That should just require building `yarn`, `pp2n` and `readlink`, right?

Reply:
Oh, I see what you mean! Yeah, good idea. :-D